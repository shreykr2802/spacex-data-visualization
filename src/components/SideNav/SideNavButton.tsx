const SideNavButton = () => {
    return (
        // eslint-disable-next-line 
        <a href="#" data-target="slide-out" className="top-nav sidenav-trigger show-on-large right" >
            <i className="material-icons">filter_list</i>
        </a>
    );
}

export default SideNavButton;