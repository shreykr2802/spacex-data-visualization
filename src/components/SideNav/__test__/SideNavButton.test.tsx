import React from 'react';
import ReactDom from 'react-dom';
import SideNavButton from '../SideNavButton';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';

afterEach(cleanup);
it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDom.render(<SideNavButton />, div);
});

it("matches snapshot", () => {
    const tree = renderer.create(<SideNavButton />).toJSON();
    expect(tree).toMatchSnapshot();
})