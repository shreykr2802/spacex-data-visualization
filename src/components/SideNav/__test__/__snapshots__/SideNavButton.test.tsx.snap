// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<a
  className="top-nav sidenav-trigger show-on-large right"
  data-target="slide-out"
  href="#"
>
  <i
    className="material-icons"
  >
    filter_list
  </i>
</a>
`;
